frontend/public/components/build-config.tsx

```diff
@@ -21,7 +21,8 @@ import {
   SectionHeading,
   WebhookTriggers,
 } from './utils';
-import { BuildsPage, BuildEnvironmentComponent, BuildStrategyType } from './build';
+import { BuildsPage, BuildEnvironmentComponent, BuildStrategyType,
+  PipelineBuildStrategyAlert } from './build';
```

Comment:
This will likely fail the linter.

Reply:
You should make sure Prettier is working in your environment.

Reply:
updated